pyomo/environ/tests/standalone_minimal_pyomo_driver.py

```diff
@@ -0,0 +1,159 @@
+import sys
+from io import StringIO
+from pyomo.common.log import LoggingIntercept
+from pyomo.common.tee import capture_output
+
+_baseline = """
```

Comment:
Not saying this should be changed, but do we want to rely on a baseline?

Reply:
It's not great, but there really isn't a better way to check a NL file (we use baselines elsewhere, and - so far - have been very stable)